pdfminer/cmapdb.py

```diff
@@ -231,14 +231,14 @@ class CMapNotFound(CMapError):
     @classmethod
     def _load_data(klass, name):
         name = name.replace("\0", "")
-        filename = '%s.pickle.gz' % name
+        filename = '%s.pickle.xz' % name
```

Comment:
The cmap files are included into the package (in the `pdfminer/cmp` directory). Changing the extension invalidates the path.